src/_igraph/graphobject.c

```diff
@@ -12902,6 +12981,39 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  multiplying by the number of vertices minus one.\n"
    "@return: the calculated closenesses in a list\n"},
 
+  /* interface to igraph_harmonic_centrality */
+  {"harmonic_centrality", (PyCFunction) igraphmodule_Graph_harmonic_centrality,
+   METH_VARARGS | METH_KEYWORDS,
+   "harmonic_centrality(vertices=None, mode=ALL, cutoff=None, weights=None,\n"
+   "          normalized=True)\n\n"
+   "Calculates the harmonic centralities of given vertices in a graph.\n\n"
+   "The harmonic centerality of a vertex measures how easily other\n"
+   "vertices can be reached from it (or the other way: how easily it\n"
+   "can be reached from the other vertices). It is defined as the\n"
+   "number of vertices minus one times the sum of one divided by \n"
+   "the length of each geodesic from/to the given vertex.\n\n"
+   "If the graph is not connected, and there is no path between two\n"
+   "vertices, the number of vertices is used instead the length of\n"
```

Comment:
"the number of vertices used" is incorrect (seems like a leftover from the old way closeness was calculated).

When there is no path between two vertices, their inverse distance is taken to be zero.

Reply:
yep, done